src/components/Example.container.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { PureComponent } from 'react'
+import styles from '../css/Example.less'
+import { Bootstrapper } from '../core/bootstrapper';
+import { Store } from '../core/store';
+
+export default class ExampleContainer extends PureComponent<any, any> {
+    render() {
+        console.log(Bootstrapper.ioc.get(Store));
+        return (
+            <div className={styles.paragraph}><span>LOADED!!!!</span></div>
```

Comment:
I think the higher order component should not contain any other logic, nor should it be a purecomponent. What do u think?